src/pages/index.tsx

```diff
@@ -150,6 +150,9 @@ const PortalHomePage = ({
 
       <EventCarousel
         title="Upcoming Events"
+        linkFilters={{
+          date: 'upcoming',
+        }}
```

Comment:
```suggestion
        linkFilters={{ date: 'upcoming' }}
```

just so that it's consistent with the other EventCarousel below